### ISSUE TITLE
Adding a toHomogeneousMatrix function to Motion

### ISSUE THREAD

**Issue by stephane-caron** [stack-of-tasks/pinocchio#1945]
Currently Pinocchio has `SE3::toHomogeneousMatrix` but not `Motion::toHomogeneousMatrix`. It arose from [this discussion](https://github.com/stack-of-tasks/pinocchio/discussions/1939) that it would be consistent and convenient to have both.

Note that in the meantime we can always do the conversions in Python:

```python
def to_homogeneous(twist: pin.Motion) -> np.ndarray: 
     homogeneous = np.zeros((4, 4)) 
     homogeneous[:3, :3] = twist.action[:3, :3] 
     homogeneous[:3, 3] = twist.linear 
     return homogeneous
```

I can help prepare a PR if this makes sense.

**Comment by jcarpent** [comment]
OK to add this method to the Motion class, even if it is not a convenient way to use Motion quantities in terms of sparsity exploitation and general efficiency.

**Comment by jcarpent** [comment]
Solved via #1946
